core/tests/batch_loader_tests.py

```diff
@@ -53,7 +53,7 @@ def test_load_batch(self):
         # extract ocr data just for this page
         loader.process_ocr(page, index=False)
         self.assertTrue(page.ocr != None)
-        self.assertTrue(len(page.ocr.text) > 0)
+        self.assertTrue(len(page.lang_text) > 0)
```

Comment:
Use [`assertGreater`](https://docs.python.org/2/library/unittest.html#unittest.TestCase.assertGreater) so if the test fails it will print the values rather than just true/false.

If this is coming from a test fixture, you could even use [`assertListEqual`](https://docs.python.org/2/library/unittest.html#unittest.TestCase.assertListEqual) to confirm the expected value.

Reply:
I added assertGreater here. Thanks